clang/test/SemaSYCL/free_function_template_negative.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -verify=expected -triple spir64-unknown-unknown -sycl-std=2020 -fsycl-int-header=%t.h %s
```

Comment:
Nether of these flags should be required to see the error. 
```suggestion
// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -verify %s
```

Reply:
Done